File: backend/api/src/main/scala/com/github/ai/leetcodequiz/apisc/response/GetQuestionnaireResponse.scala
package com.github.ai.leetcodequiz.apisc.response

import com.github.ai.leetcodequiz.apisc.QuestionnaireItemDto
import zio.json.{JsonDecoder, JsonEncoder}

case class GetQuestionnaireResponse(
  questionnaire: QuestionnaireItemDto
) derives JsonEncoder, JsonDecoder

File: android/backend-api/src/main/kotlin/com/github/ai/leetcodequiz/api/response/GetQuestionnaireResponse.kt
package com.github.ai.leetcodequiz.api.response

import kotlinx.serialization.Serializable
import com.github.ai.leetcodequiz.api.QuestionnaireItemDto
import com.github.ai.leetcodequiz.api.QuestionnairesItemDto

@Serializable
data class GetQuestionnaireResponse(
    val questionnaire: QuestionnaireItemDto
)